File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int>s(nums.begin(),nums.end());
        int m=s.size();
        int i=0;
        for(int x:s){
            nums[i++]=x;
        }
        return m;
        
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        s=set()
        n= len(nums)
        for i in range(0,n):
            s.add(nums[i])
        m=len(s)
        unique = sorted(s)   
        nums[:] = unique  
        return m
        